gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <h2 className="h2">Page specific stylesheets</h2>
+    <p className="nx-p">
+      Take a look at <code className="nx-code">frontend/src/main/scss/app-styles.scss</code>. This file
+      contains custom styles for the app and imports external (page specific) stylesheets. Why have page
+      specific stylesheets? By grouping our styles together by use and (ideally) locating the SCSS
+      next to the HTML that it modifies we decrease clutter/noise in our main stylesheet, keep our styles focused
+      to what they modify, and also help maintain a clear separation between code, layout, and styling.
+    </p>
+    <h2 className="h2">B.E.M.</h2>
+    <p className="nx-p">
+      You might have noticed lots of dashes and underscores in our class names. That's because we use BEM naming
+      by default. BEM stands for "Block", "Element", and "Modifier".
+      <a href="https://www.toptal.com/css/introduction-to-bem-methodology">Learn more about BEM</a>. We use a
```

Comment:
On the Additional Resources PR, all of the links open in a new tab.  Should we do that here too?

Reply:
> On the Additional Resources PR, all of the links open in a new tab. Should we do that here too?

Good practice for all external link IMO: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/1601290a56c5aaced42bf72f9d2a2a1baef18d24

Reply:
> Good practice for all external link IMO: 1601290

Don't forget `rel="noreferrer"`